File: Facebook.NET/GraphAPI.Domain.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphAPI
    {
        [DataContract]
        public sealed class Domain : IObject
        {
            [DataMember(Name = "id")]
            public string ID { get; set; }

            [DataMember(Name = "name")]
            public string Name { get; set; }
        }
    }
}

File: Facebook.NET - VB/Contracts/GraphAPI.Domain.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class Domain
        Implements IObject
            <DataMember(Name:="id")>
            Public Property ID() As String Implements IObject.ID

            <DataMember(Name:="name")>
            Public Property Name() As String Implements IObject.Name
        End Class
    End Class
End Namespace
